src/service/extensions-impl.js

```diff
@@ -205,10 +205,9 @@ export class Extensions {
    * @param {string} extensionId
    * @param {function(!Object)} factory
    * @param {!Object} arg
-   * @private
```

Comment:
Mark these `@protected` since it's only used within the file. (Assuming this works, it might make sense to include it in your other in-flight PRs.)

https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#protected

Reply:
We could do `@protected`, but that's not what it means to me.

For context, `@private` is also accessible to [anything in the same file](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#private), but that conflicts with the direction JS is taking with private fields.

Reply:
Or, we already have uses of `@restricted`.

Reply:
SGTM.